co2calculator/calculate.py

```diff
@@ -54,6 +54,9 @@ def calc_co2_car(
     :return: Total emissions of trip in co2 equivalents, total distance of the trip
     :rtype: tuple[float, float]
     """
+    # NOTE: Tests show very little emissions for diesel cars (as low as electric)!
+    # NOTE: Tests fail for 'cng'  as `fuel_type` (IndexError)
```

Comment:
https://github.com/pledge4future/co2calculator/blob/dev/data/emission_factors.csv
Yes, it seems like we do not have the combo of 'cng' for fuel type and 'average' for the size.. maybe it would be a good idea to add an entry for 'average' size and then just use the emission factor for 'medium' 'cng' cars. No idea what's the issue with the 'diesel' car though, the emission factors in the table seem accurate..

Reply:
@akokam 
I investigated this and there was a major bug in the code.. we did not query for the categories/subcategories so the function for car emissions for example used the emission factors for a diesel bus (instead of diesel car).
Thanks a lot for noticing - and see you soon 😉